components/MapList/index.js

```diff
@@ -64,6 +66,7 @@ function MapList({
   title,
   getMapIcon,
   featurePk,
+  noFeaturesMessage,
```

Comment:
i love this prop and think it is a great solution 🙏 

Would you add an `@property` docstring about it, and also add it to the `MapList.propTypes` down below?